src/index.html

```diff
@@ -4,9 +4,346 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header" >
+      <a class="header__logo-link" href="#">
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <div class="card__image">
+            <img
+              class="img"
+              src="images/imac.jpeg"
+              alt="Image product imac"
+            >
+          </div>
```

Comment:
```suggestion
          <div class="card__image-container">
            <img
              class="card__image"
              src="images/imac.jpeg"
              alt="Image product imac"
            >
          </div>
```
Let's follow to BEM style naming